M3T/CMakeLists.txt

```
@@ -32,7 +33,14 @@ if (UNIX)
 endif ()
 find_package(OpenGL REQUIRED)
 find_package(glfw3 3.1.2 REQUIRED)
-find_package(OpenCV 4.3.0 REQUIRED COMPONENTS core imgproc highgui imgcodecs calib3d features2d xfeatures2d OPTIONAL_COMPONENTS cudafeatures2d)
+
+set(OpenCV_COMPONENTS core imgproc highgui imgcodecs calib3d features2d xfeatures2d)
```

```suggestion
set(OpenCV_COMPONENTS core imgproc highgui imgcodecs calib3d features2d)
```
Shouldn't it be like this?